shell/platform/linux/fl_view.cc

```diff
@@ -43,21 +49,22 @@ struct _FlView {
   FlMouseCursorPlugin* mouse_cursor_plugin;
   FlPlatformPlugin* platform_plugin;
   FlTextInputPlugin* text_input_plugin;
+  FlPlatformViewsPlugin* platform_views_plugin;
+
+  FlGestureHelper* gesture_helper;
 
   GList* gl_area_list;
   GList* used_area_list;
 
+  std::map<GtkWidget*, FlClippingView*>* current_clipping_views;
+  std::map<GtkWidget*, FlClippingView*>* last_clipping_views;
+
   GtkWidget* event_box;
 
-  GList* children_list;
-  GList* pending_children_list;
+  std::vector<GtkWidget*>* children_list;
+  std::vector<GtkWidget*>* pending_children_list;
```

Comment:
Please keep these as GList (or GPtrArray if that is required).

Reply:
Done.